src/kibana/components/vislib/components/tooltip/tooltip.js

```diff
@@ -86,12 +86,22 @@ define(function (require) {
         selection.each(function (d, i) {
           var element = d3.select(this);
 
+          /*
+           * Renders the tooltip.
+           *
+           * @param html {optional Boolean, String} - if HTML, or some string is passed in,
+           * this will be rendered. If False is passed in the tooltip will be hidden.
+           */
           function render(html) {
             allContents = _.filter(allContents, function (content) {
               return content.id !== id;
             });
 
-            if (html) allContents.push({ id: id, html: html, order: order });
+            if (html) {
+              allContents.push({ id: id, html: html, order: order });
+            } else if (html === false) {
+              allContents = [];
```

Comment:
I don't this is an appropriate fix. The purpose of the `allContents` array is to track the contents of all tooltip instances. If a single `mousout` event shouldn't clear the contents of all elements.